ckanext/security/logic/auth.py

```diff
@@ -13,3 +13,6 @@ def security_throttle_user_show(context, data_dict):
 
 def security_throttle_address_show(context, data_dict):
     return {'success': False}
+
+def security_reset_totp(context, data_dict):
+    return {'success': False}
```

Comment:
Does this mean this endpoint will only work for sysadmins?

Reply:
Yes. Sysadmins bypass auth according to the docs so if we just reject all auth its only available to sysadmins.